pages/bilder/_id.vue

```diff
@@ -17,15 +17,14 @@
             </v-btn>
           </p>
           <h2 class="text-center ichen_blue--text text-h2 pt-0 pb-3">
-            {{ bilderstrecke.Titel }}
+            {{ bilderstrecke.titel }}
           </h2>
           <v-carousel hide-delimiters>
             <v-carousel-item
-              v-for="Bild in bilderstrecke.Bilder"
-              :key="Bild.id"
+              v-for="bild in bilderstrecke.bilder"
+              :key="bild.id"
               contain
-              :src="Bild.url"
-              :srcset="Bild | generateSrcset"
```

Comment:
For simplicity, I removed `generateSrcset` - this is a TODO. We have to find a solution here for Nuxt and HyGraph.